pkg/sqlcache/informer/informer.go

```diff
@@ -34,15 +34,21 @@ var newInformer = cache.NewSharedIndexInformer
 
 // NewInformer returns a new SQLite-backed Informer for the type specified by schema in unstructured.Unstructured form
 // using the specified client
-func NewInformer(client dynamic.ResourceInterface, fields [][]string, transform cache.TransformFunc, gvk schema.GroupVersionKind, db sqlStore.DBClient, shouldEncrypt bool, namespaced bool) (*Informer, error) {
+func NewInformer(client dynamic.ResourceInterface, fields [][]string, transform cache.TransformFunc, gvk schema.GroupVersionKind, db sqlStore.DBClient, shouldEncrypt bool, namespaced bool, watchable bool) (*Informer, error) {
+	watchFunc := func(options metav1.ListOptions) (watch.Interface, error) {
+		return client.Watch(context.Background(), options)
+	}
+	if !watchable {
+		watchFunc = func(options metav1.ListOptions) (watch.Interface, error) {
+			return newSyntheticWatcher().watch(client, options, 5*time.Second)
```

Comment:
This seems to be a reasonable default.

I recommend extracting it into a constant at the top of the file, in case we need to tune it (eg. by GVK) at some point.

Reply:
Well, a variable with that default value and a setter to call to modify it if necessary.